components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -145,7 +147,7 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
     }
 
     public render() {
-        const {id, postId, text, selectedValueText, subMenu, icon, filter, ariaLabel, direction, styleSelectableItem} = this.props;
+        const {id, postId, text, selectedValueText, subMenu, icon, filter, ariaLabel, direction, styleSelectableItem, renderLeft, renderRight} = this.props;
```

Comment:
didn't @matthewbirtch say we will always render the chevron on the right side as an indicator that there is a submenu, not to show in which direction the submenu opens?

Reply:
Yes, I did :)

Reply:
yup this change should render it to the right... or you mean remove the option to render to the left? I thought we can keep it just in case but I can also remove it :)